_episodes/02-numpy.md

```diff
@@ -546,6 +546,20 @@ which is the average inflammation per patient across all days.
 > > Creates a substring from index 1 up to (not including) the final index,
 > > effectively removing the first and last letters from 'oxygen'
 > {: .solution}
+>
+> How can we rewrite the slice for getting the last three characters of `element` does,
+> so it will work on any character string at least three characters in length.
```

Comment:
Can you remove the word "does" on line 550 and change the period (`.`) to a question mark (`?`) at the end of line 551?